bin/travis-deploy

```diff
@@ -20,14 +38,16 @@ This is an automated update by travis-ci at
   make upload-cache
 }
 
-if git diff --exit-code --quiet; then
+if [ "${TRAVIS_EVENT_TYPE}" = "pull_request" ]; then
```

Comment:
I feel like this check should be moved down, after the `git diff` check. Otherwise you might be diffing an empty `build` dir or posting a diff when there are no changes, right?

Reply:
Yeah - that behavior is by design. I'd rather see a comment with no diff (to know everything's working) than not see a comment if there's no diff.